src/validatedODEs.jl

```diff
@@ -573,14 +589,14 @@ function validated_integ(f!, X0, t0::T, tmax::T, orderQ::Int, orderT::Int, absto
                                 nsteps, orderT, red_abstol, params, 
                                 parse_eqs, adaptive, minabstol, 
                                 absorb, check_property)
-        δtI = sign_tstep * Interval{T}(0, sign_tstep*δt)
+        δtI = sign_tstep * Interval(zt, sign_tstep*δt)
 
-        # New initial conditions and time
+        # New initial conditions and time, and output vectors
         nsteps += 1
+        @inbounds tv[nsteps] = t0
```

Comment:
@lbenet was this change intended? note that `tb[nsteps]` came *after* updating `t0 += δt`. so the result is now different. seems like a breaking change (see https://github.com/JuliaReach/ReachabilityAnalysis.jl/pull/485/checks?check_run_id=2690783070#step:6:347)

Reply:
to give a concrete example, consider:

```julia
@taylorize function f!(dx, x, p, t)
    dx[1] = x[2]
    dx[2] = zero(x[2])
    return dx
end

# Initial conditions
tini, tend = 0.0, 0.1
X0 = interval(0.95, 0.99) × interval(0.1, 0.1)

# Parameters
abstol = 1e-10
orderQ = 2
orderT = 4
ξ = set_variables("ξₓ ξᵥ", order=2*orderQ, numvars=length(q0))

tTM, qv, qTM = validated_integ(falling_ball!, X0, tini, tend, orderQ, orderT, abstol);

@show tTM;
@show domain(qTM[:, 1][1]);
@show domain(qTM[:, 2][1]);

tTM = [0.0, 0.0]
domain((qTM[:, 1])[1]) = Interval(0.0, 0.0)
domain((qTM[:, 2])[1]) = Interval(0.0, 0.1)
```
here i expected the second component of `tTM` to be 0.1.